django_cloud_deploy/integration_tests/workflow_test.py

```diff
@@ -246,3 +248,49 @@ def test_deploy_new_app_sync(self):
                     secrets=secrets)
                 response = requests.get(admin_url)
                 self.assertIn('Django administration', response.text)
+
+
+class ProjectWorkflowIntegrationTest(test_base.BaseTest):
```

Comment:
Theres no test for positive creation?

Reply:
Yeah. The way our integration test works does not allow creation of new GCP projects,